src/components/stories/stories-wrapper.tsx

```diff
@@ -7,26 +7,38 @@ import {StoriesLoaderItem} from '@app/components/stories/stories-loader-item';
 import {StoriesPreviewItem} from '@app/components/stories/stories-preview-item';
 import {createTheme} from '@app/helpers';
 import {Stories} from '@app/models/stories';
+import {IStory} from '@app/types';
 
-export const StoriesWrapper = observer(() => {
-  const content = useMemo(() => {
+type Props = {
+  onStoryPress: (id: IStory['id']) => void;
+};
+
+export const StoriesWrapper = observer(({onStoryPress}: Props) => {
+  const Content = memo(() => {
```

Comment:
add useCallback

Reply:
Pretty much the same behaviour as memo HOC

Reply:
Thanks! I found out, that memo HOC memoize only function call, not the creation (render, not creation). Fixed